fast/stages/2-networking-a-simple/data/firewall-policy-rules/dev/ingress.yaml

```diff
@@ -0,0 +1,10 @@
+# skip boilerplate check
+
+ingress-default-deny:
+  description: "Deny and log any unmatched ingress traffic."
+  priority: 2147483643
+  action: "deny"
+  match:
+    source_ranges:
+      - "0.0.0.0/0"
+  enable_logging: true
```

Comment:
This doesn't seem to exactly have the effect of what we were doing in vpc firewall rules. Thoughts?